src/culc/TestCulc.java

```diff
@@ -0,0 +1,138 @@
+package culc;
+
+import java.util.Scanner;
+
+public class TestCulc {
+
+    public static String[] romanNumber = {"I", "II", "III", "IV", "V", "VI", "VII", "VIII", "IX", "X", "XX", "XXX", "XL", "L",
```

Comment:
Better to create separate arrays for such group as (1-9), (10-90), (100-1000) etc